docs/api-reference/endpoints/workspaces/get-workspace-id.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: "Get Project ID"
```

Comment:
May be its better we say `Get Project by Slug`

Reply:
hmmm main intention of this endpoint is to just fetch the ID of the project...  are you implying we should just make it fetch the entire project? so we support fetching project by ID, name, and slug?